lib/aur-build

```diff
@@ -203,6 +202,11 @@ elif ! [[ -w $db_path ]]; then
     exit 13
 fi
 
+# Custom makepkg command
+if [[ ${MAKEPKG-} ]]; then
```

Comment:
although this seems to work, is it the same as using `${MAKEPKG:-}`, and if so, lets maybe use this syntax instead as it is more common? unless `${MAKEPKG-}` has a different behavior, which I can't find documentation for...

Reply:
The difference between `-` and `:-` is that the value following it would be substituted if `$MAKEPKG`is empty, or empty and unset, respectively. It doesn't really matter in this case, I only added the `-` if the script is run with `bash -u` and we could leave it out entirely.

Reply:
Ah I see, interesting - I don't mind either way, leave whatever you prefer, just interesting to know there is an actual difference 👍